spec/label_spec.rb

```diff
@@ -0,0 +1,89 @@
+require_relative '../classes/Book/label'
+require_relative '../classes/item'
+
+describe Label do
+  describe 'unit tests for \'Label\' class' do
+    before(:each) do
+      # set the params to create a Label instance
+      @label_params_req = {
+        name: 'test label',
+        title: 'book label',
+        color: 'red'
+      }
+      @label_params_id = {
+        id: 1,
+        name: 'test label'
+      }
+      @new_label_req = Label.new(@label_params_req)
+      @new_label_id = Label.new(@label_params_id)
+
+      @label_instance = instance_double(Label)
+
+      # create an Item instance
+      item_params_id = {
+        id: 1,
+        publish_date: '2023-08-01'
+      }
+      @new_item = Item.new(item_params_id)
+    end
+
+    context 'when creating a new \'Label\'' do
+      it 'should create an instance of \'Label\' class' do
+        expect(@new_label_req).to be_an_instance_of Label
+        expect(@new_label_id).to be_an_instance_of Label
```

Comment:
- Brackets? or not? Like in lines 39 and 40.
```suggestion
        expect(@new_label_req).to be_an_instance_of(Label)
        expect(@new_label_id).to be_an_instance_of(Label)
```
- Just to keep the same style.